src/supported-value.js

```diff
@@ -64,7 +64,7 @@ export default function supportedValue(property, value) {
   // IE can even throw an error in some cases, for e.g. style.content = 'bar'.
   try {
     // Test value as it is.
-    el.style[property] = prefixedValue
+    el.style.setProperty(property, ...prefixedValue.split(/!(?=important$)/))
```

Comment:
1. I think we don't need to pass important for the test at all, because the result for our tests should be the same no matter if there is important or not
2. we need to do some micro benchmarking on esbench to see what's the fastest way to get rid of important, I would guess str.replace will be much faster than doing the split

Reply:
My guess is this hsould be the fastest: `.replace(/!important[\s]*$/, '')`  - I think supporting accidental whitespace at the end is a good idea